### DISCUSSION TITLE
Limit exceeded as python dependecy modules are added

### DISCUSSION THREAD

**Discussion by vrc29797** [deta/space-docs#144] [Q&A]
Output size limit exceeded: 0.00 MB > 262.14 MB

my  requirements.txt

discord
requests
python-dotenv
Pillow
rembg

**Comment by lihugang** [comment]
maybe your dependencies are too large that exceeds the limit?

**Reply by vrc29797** [reply]
yes, That is the case, but how to reduce it, so it gets under the limit?

**Reply by lihugang** [reply]
here are some solutions which may work.
https://github.com/orgs/deta/discussions/139#discussioncomment-6331574
https://www.reddit.com/r/aws/comments/4qrw9m/how_to_work_around_aws_lambdas_250mb_limit/

**Reply by lihugang** [reply]
1. try to use smaller packages instead
2. split your codes and dependencies to different micros, use http requests to transfer messages
3. pack your application and use `engine: custom` to execute the executable file (I know little about Python, I don't sure whether this tip works)
